app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -26,7 +26,7 @@
 
 <div class="govuk-grid-row app-print-link app-!-print-display-none">
   <div class="govuk-grid-column-two-thirds">
-    <%= govuk_link_to('Print this page', 'javascript:window.print();', class: 'app-print-link__link') %>
+    <%= govuk_link_to('View as PDF', provider_interface_application_choice_path(@application_choice.id, format: :pdf)) %>
```

Comment:
1. Let's put it in the top right of the page in line with the h1
2. Can you add a `download` attribute on the link so it doesn't open the PDF in the browser
3. I'll work on the text for the link and put it in the prototype which may help you with (1)

Reply:
@adamsilver What's the rationale of downloading rather than opening in the browser? Wouldn't we want to offer the user the option to print directly from an embedded PDF viewer rather than download, then locate and open a file, then print?

Reply:
The rationale is that clicking ‘Download X’ should ‘Download’ rather than show X. It would be slightly deceptive to open it.

It also means they're free to open the PDF in an application of their choosing, which I think will be something alternative to the browser.